doc/en/tmpdir.rst

```diff
@@ -5,6 +5,51 @@
 Temporary directories and files
 ================================================
 
+The ``tmp_path`` fixture
+----------------------
+
+.. versionadded:: 3.9
+
+
+You can use the ``tmpdir`` fixture which will
+provide a temporary directory unique to the test invocation,
+created in the `base temporary directory`_.
+
+``tmpdir`` is a `pathlib/pathlib2.Path`_ object. Here is an example test usage::
+
+    # content of test_tmp_path.py
```

Comment:
I believe it is better to use `.. code-block:: python` so that blaken-docs can do its thing.